### ISSUE TITLE
Extensibility of claims' semantics

### ISSUE THREAD

**Issue by thomas-fossati** [ietf-rats-wg/eat#74]
If I wanted to expand the semantics of "Debug Disable" or "Security Level" with my own local semantics, how would I do that?

Should I create a new claim?  Or could I extend the claim’s type?  If the latter, how? 

It’d be useful that the doc explicitly stated the rules of this game.

**Comment by thomas-fossati** [comment]
Laurence [comment on the mailing list](https://mailarchive.ietf.org/arch/msg/rats/DVCR2E_Dozx0JhHz9Pd5w3RdADU/) :
```
I don’t think these should be extensible. You can’t make up
your own corporate bond rating category or restaurant hygene
rating category and such. If that is allowed the scheme becomes
much less useful. 

I’ve added text for that in this PR. It says that when you can’t live
with the defined security level schema, omit the claim, make up
your own claim or do both.

Also, see this old but unresolved discussion on security level.
One thing nice about a very simple three-level scheme is that it
becomes clear this is a very rough top-level characterization only.
```

**Comment by laurencelundblade** [comment]
Addressed with #82 and #73
